examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1436,18 +1466,24 @@ def compute_text_embeddings(prompt, text_encoders, tokenizers):
                 # (this is the forward diffusion process)
                 noisy_model_input = noise_scheduler.add_noise(model_input, noise, timesteps)
 
+                # time ids
+                add_time_ids = torch.cat(
+                    [
+                        compute_time_ids(original_size=s, crops_coords_top_left=c)
```

Comment:
(**nit**): `compute_time_ids()` takes arguments in the following order: crops_coords_top_left -> original_size. Since the function spec is short enough, I think it makes sense to provide them in this order. WDYT?